api/tests/DataPersister/Util/DataPersisterObservableTest.php

```diff
@@ -154,10 +154,10 @@ public function testCallCustomActionListenersForItemOperation() {
         $this->dataPersisterObservable->persist($toPersist, ['item_operation_name' => self::ACTION_NAME]);
     }
 
-    public function testCallCustomActionListenersForItemOperationTwiceIfRegisteredTwice() {
+    public function testCallCustomActionListenersForItemOperationOnlyOnceIfRegisteredTwice() {
```

Comment:
Okay, we share the DataPersisterObservable between two requests?

Reply:
Yes, in the tests when reboot is enabled with:
```
$client->disableReboot();
```